packages/gafl-webapp-service/src/pages/licence-details/licence-start-time/__tests__/licence-start-time.spec.js

```diff
@@ -76,7 +76,9 @@ describe('The licence start time page', () => {
     })
 
     const minHour = moment()
+      .add(1, 'hour')
```

Comment:
Not sure I understand why we're adding an hour here?  Should this be using moment-timezone instead?

Reply:
const minHour = moment()
      .add(1, 'hour')
      .add(30, 'minute')
      .startOf('hour')
      .hour()
If you are starting a licence at a given time of day, today the minimum hour (local time) you can choose to start the licence is calculated as above.

So for instance 8:29am goes -> 9:29 -> 9:59 -> 9:00 -> 9am radio button is enabled
and 8:31 -> 9:31 -> 10:01 -> 10:00 -> 10am radio button is enabled.

Reply:
👍  that makes sense.  shouldn't this be using moment-timezone though?